src/scitokens/scitokens.py

```diff
@@ -170,16 +171,23 @@ def serialize(self, include_key=False, issuer=None, lifetime=600):
         issue_time = int(time.time())
         exp_time = int(issue_time + lifetime)
         
-        payload = dict(self._claims)
+        # Add to validated and other claims
+        payload = dict(self._verified_claims)
+        payload.update(dict(self._claims))
```

Comment:
Casting to `dict` isn't necessary, is it?

Reply:
The Second isn't necessary.  But the first is done so we can "copy" the _verified_claims dict and not make modifications to it.